### PR TITLE
SDI-477: ✨ Switch to open telemetry

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt [hunk 1]
```diff
@@ -4,8 +4,8 @@ import org.gradle.api.Project
 import org.gradle.api.tasks.Copy
 import uk.gov.justice.digital.hmpps.gradle.ConfigManager
 
-private const val APP_INSIGHTS_SDK_VERSION = "2.6.4"
-private const val APP_INSIGHTS_AGENT_VERSION = "3.4.7"
+private const val APP_INSIGHTS_VERSION = "3.4.7"
+private const val OPEN_TELEMETRY_VERSION = "1.21.0"
```
Comment (neilmendum) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt:8]: As these need to be bumped manually we could include a comment to the releases pages for them, to make it easier for folks to check the latest versions?

I think the links are
- https://github.com/microsoft/ApplicationInsights-Java/releases
- https://github.com/open-telemetry/opentelemetry-java/releases

Comment (petergphillips) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt:8]: although they need to be bumped manually, the latest versions can be found for these (and all other manual dependencies) by following the instructions in https://github.com/ministryofjustice/dps-gradle-spring-boot#how-to-check-if-dependencies-applied-by-the-dps-plugin-are-up-to-date

Comment (neilmendum) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt:8]: Didn't know about that, neat!

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**